index.js

```diff
@@ -51,11 +61,9 @@ const parseArgs = (
   argv = getMainArgs(),
   options = {}
 ) => {
-  if (typeof options !== 'object' || options === null) {
-    throw new Error('Whoops!');
-  }
-  if (options.withValue !== undefined && !ArrayIsArray(options.withValue)) {
-    throw new Error('Whoops! options.withValue should be an array.');
+  validateObject(options, 'options');
+  if (options.withValue !== undefined) {
```

Comment:
if we care about "present and undefined", then it seems like null should be ignored too - otherwise, could we use hasOwnProperty here?

Reply:
I was thinking it might simplify things to just use `ObjectHasOwnProperty` too.

Reply:
👍 
`ObjectHasOwnProperty` seems the right tool for the job. (I wondered about validateObject with custom options, but still have the annoying defined-as-undefined, which is fresh in my mind from working on parse error cases!)

Reply:
done.